bender-sim.mk

```diff
@@ -8,7 +8,9 @@
 # bender targets
 sim_targs += -t sim
 sim_targs += -t test
+sim_targs += -t snitch_cluster
```

Comment:
This should go in the bender-common.mk since it has to be shared for synthesis, FPGA and RTL simulation.

Reply:
fixed